js/src/Axis.ts

```diff
@@ -112,9 +118,9 @@ export class Axis extends WidgetView {
         } else if (num_ticks !== undefined && num_ticks !== null) {
             this.axis.tickValues(this.get_ticks_from_array_or_length());
         } else {
-            if (this.axis_scale.model.type === "ordinal") {
+            if (this.axis_scale instanceof OrdinalScale) {
```

Comment:
Is this going to work? `axis_scale` is still of type `Scale`:
https://github.com/davidbrochart/bqplot/blob/5c9ad992431b8251b79f1210c597ff245508d56c/js/src/Axis.ts#L925

Reply:
Yes. An `OrdinalScale` is a `Scale` :)